borsh-derive/src/internals/attributes/item/mod.rs

```diff
@@ -86,18 +92,22 @@ pub fn contains_use_discriminant(input: &ItemEnum) -> Result<bool, syn::Error> {
 }
 
 pub(crate) fn contains_initialize_with(attrs: &[Attribute]) -> Option<Path> {
+    let mut res = None;
     for attr in attrs.iter() {
-        if attr.path() == INIT {
-            let mut res = None;
+        if attr.path() == BORSH {
```

Comment:
I would rewrite the loop into iter().find() to make it easier to reason about

Reply:
Ok. Thanks!